xml/depl_crowbar_extra-features.xml

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE chapter
+[
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec.depl.adm_crowbar_extra-features">
+ <title>Customizing &crow;</title>
+  <para>
+   In large deployments with many nodes, there are always some nodes that are
+   in a fail or unknown state. New &barcl;s cannot be applied to them and
+   values cannot be updated in some &barcl;s that are already deployed. This
+   happens because &crow; will refuse to apply a &barcl; to a list of nodes if
+   they are not all in <literal>ready</literal> state.
+  </para>
+  <para>
+   To avoid having to manually take out nodes that are not
+   <literal>ready</literal>, there is a feature called <literal>skip
+   unready nodes</literal>. Instead of refusing to apply the &barcl;, it will
+   skip the nodes that it finds in any other state than
+   <literal>ready</literal>.
+  </para>
+  <para>
+   <emphasis role="bold">Enabling the Feature</emphasis>
+  </para>
+  <para>
+   In <filename>/opt/dell/crowbar_framework/config/crowbar.yml</filename> set
+   the option <literal>skip_unready_nodes</literal> to <literal>true</literal>.
+  </para>
+  <screen>default: &amp;default
+skip_unready_nodes:
+  enabled: false &lt;&lt;&lt; change to true</screen>
+  <para>
+   <emphasis role="bold">Roles Affected</emphasis>
+  </para>
+  <para>
+   All &Barcl; roles are affected. The default config file includes all the
+   roles that have been tested and found to be working. Adding roles to the
+   default list is not supported for the <literal>skip_unready_nodes</literal>
+   feature. Removing default roles is supported.
+  </para>
+  <para>
+   The list of currently supported roles to skip:
+  </para>
+  <screen>- bmc-nat-client
+- ceilometer-agent
+- deployer-client
+- dns-client
+- ipmi
+- logging-client
+- nova-compute-ironic
+- nova-compute-kvm
+- nova-compute-qemu
+- nova-compute-vmware
+- nova-compute-xen
+- nova-compute-zvm
+- ntp-client
+- provisioner-base
+- suse-manager-client
+- swift-storage
+- updater</screen>
+  <para>
+   <emphasis role="bold">Determining Which Nodes Were Skipped</emphasis>
+  </para>
+  <para>
+   Skipped nodes are logged to the &crow; log
+   (<filename>/var/log/crowbar/production.log</filename>) where you can search
+   for the text <literal>skipped until next chef run</literal>. This
+   will print the log lines where nodes were skipped, the name of the node, and
+   the &barcl; which was being applied.
+  </para>
+  <screen>&prompt.user;grep "skipped until next chef run" /var/log/crowbar/production.log</screen>
+  <para>
+   You can also search the production log for skipped unready nodes.
+  </para>
+  <screen>&prompt.user;grep "skip_unready_nodes" /var/log/crowbar/production.log</screen>
+  <para>
+   After enabling/disabling the <literal>skip_unready_nodes</literal> feature
```

Comment:
Can we wrap this section in one of those `Important` yellow boxes as to highlight it?

I think its important for the end user to be aware of that part.

Reply:
Important tag around 
   After enabling/disabling the <literal>skip_unready_nodes</literal> feature
   or adding/removing roles, the &crow; framework service must be restarted
   (<command>systemctl restart crowbar</command>) in order to use the updated
   settings.